lib/matplotlib/axes/_base.py

```diff
@@ -4255,41 +4255,14 @@ def _set_view(self, view):
         self.set_xlim((xmin, xmax))
         self.set_ylim((ymin, ymax))
 
-    def _set_view_from_bbox(self, bbox, direction='in',
+    def _get_view_from_bbox(self, bbox, direction='in',
```

Comment:
Perhaps that would be better named something like `_prepare_view_from_bbox`/`_compute_view_from_bbox`?  (Because this isn't the getter (in the usual sense) that goes with `_set_view_from_bbox.)

Reply:
Agreed, I like "prepare" and updated accordingly.